src/Tribe/Template/Month.php

```diff
@@ -487,24 +551,24 @@ public static function day_classes() {
 			}
 
 			list ( $year, $month, $day ) = explode( '-', $calendar_day['date'] );
-			if ( self::$current_month == $month && self::$current_year == $year ) {
+			if ( $current_month == $month && $current_year == $year ) {
 				// Past, Present, Future class
-				if ( self::$today == $day ) {
+				if ( $today == $day ) {
 					$ppf .= ' tribe-events-present';
 				} else {
-					if ( self::$today > $day ) {
+					if ( $today > $day ) {
```

Comment:
These `if`/`else` statements can be compressed into `ifelse` blocks if you wanted (not required).